qutip_lattice/lattice.py

```diff
@@ -45,6 +43,65 @@
     pass
 
 
+from scipy import sparse
+from scipy.sparse.linalg import eigs
+import operator as op
+from functools import reduce
+import copy
+from numpy import asarray
+import math
+
+
+__all__ = ['cell_structures', 'Lattice1d', 'Lattice1d_fermions', 'n_i',
+           'Lattice1d_bose_Hubbard', 'Lattice1d_2c_hcb_Hubbard', 'TransOperator',
+           'Lattice1d_fermi_Hubbard', 'Lattice1d_hardcorebosons', 'ncr', 'Is',
+           'Sx', 'Sy', 'Sz', 'I', 'b_', 'bd', 'commutator', 'anticommutator',
+           'a_', 'ad', 'opow', 'oprd', 'osum']
+
+def Is(i, levels=2): return [qeye(levels) for j in range(0, i)]
```

Comment:
All these utilities functions should be properly documented if made public.
If private, please add a one line docstring at least since the names are not always sufficient to understand what they do.